File: Java/BancoMain.java
public class BancoMain {
    public static void main(String[] args) {
        // Cria um novo cofrinho
        Banco meuBanco = new Banco();
        meuBanco.ContaEmBanco("Marcus Schebek");

        // Realiza operações de depósito e saque
        meuBanco.Deposita(100);
        meuBanco.Saca(30);

        // Verifica e exibe o saldo atual
        System.out.println("Saldo atual do Banco de " + meuBanco.GetName() + ": R$ "  + meuBanco.GetSaldo());
    }
}

File: C++/BancoMain.cpp
#include <iostream>
#include "Banco.h"

int main (){
    Banco meuBanco;
    meuBanco.ContaEmBanco("Marcus Schebek");

    meuBanco.Deposita(100);
    meuBanco.Saca(30);

    // Verifica e exibe o saldo atual
    float saldoAtual = meuBanco.getSaldo();
    string nome = meuBanco.getNome();
    std::cout << "Saldo atual do banco de " << nome << " : R$ "  << saldoAtual << std::endl;

}